rest/v10/index.ts

```diff
@@ -609,6 +609,15 @@ export const Routes = {
 		return `/voice/regions` as const;
 	},
 
+	/**
+	 * Route for:
+	 * - GET `/guilds/{guild.id}/voice-states/@me`
+	 * - GET `/guilds/{guild.id}/voice-states/{user.id}`
+	 */
+	voiceState(guildId: Snowflake, userId: Snowflake | '@me' = '@me') {
```

Comment:
Shouldn't we be reusing the existing [`guildVoiceState()`](https://github.com/discordjs/discord-api-types/blob/main/rest/v10/index.ts#L793) function for this instead? That function uses the same route as this, and would only require additional JSDoc comments to indicate it now supports the GET method as well.

Reply:
You're right, I missed this. Though voice state endpoints have been moved to a separate category altogether (moved from Guild) in the API documentation, so I'm wondering if we should still call it `guildVoiceState`

Reply:
Ig it doesn't make much sense renaming it.